### DISCUSSION TITLE
Does editing metadata update the original file?

### DISCUSSION THREAD

**Discussion by JamesSwift** [photoprism/photoprism#1063] [Q&A] [answered]
Hi, just a quick question. I've browsed around the docs but not managed to find a definite answer. What exactly happens when you edit a photo through PP? Which files are changed?

If I were to reimport my library to a fresh PP instance, would it see the saved changes?

Also, does changing the date on a file move the file to the correct dated folder in `originals`?

**Comment by lastzero** [comment, marked as answer]
No, we don't modify originals.

**Reply by JamesSwift** [reply]
Thanks for the clarification. So, if I wanted to move my library to a new instance by importing the photos again, how would I get the sidecar files included?

**Reply by lastzero** [reply]
Copy them as well, to the same folders :)

**Reply by JamesSwift** [reply]
Ok, so just merge the storage/sidecar folder on top of the originals folder? Hypothetically...

**Reply by lastzero** [reply]
Don't overwrite existing files, should be safe then.

**Comment by JamesSwift** [comment]
> Also, does changing the date on a file move the file to the correct dated folder in `originals`?

You might have guessed my reason for asking about re-importing was to do with this. Once Batch UX lands, I plan to get my Library all tidied up. I'd like at the end of it to have all my photos in the proper directory for when they were taken, rather than the directory they just happened to end up in.

Is the best way to achieve this to finish editing all my meta-data, then reimport the whole things into a fresh instance as described above?

Would a a feature request to optionally automatically move images into the right folder once their date-taken has been modified be likely to ever be implemented, or is it too out there? Just don't want to clog up the issue tracker.

**Reply by graciousgrey** [reply]
We consider to enable users to use PhotoPrism to physically move their files between folders later. When we come to this we could think about settings like automatically sort into folders by date taken. But it will take a while ;)
